scripts/private/_send-packet-config.js

```diff
@@ -8,43 +8,46 @@ if (!source) {
   process.exit(1);
 }
 
-function runSendPacket(config) {
+function runSendPacketCommand(command) {
+  console.log("Running sendPacket command: ", command);
```

Comment:
```suggestion
  // Console logging for debugging
  // console.log("Running sendPacket command: ", command);
```

Reply:
Think it makes sense to keep it in for debugging, but would prefer to comment out for regular usage to abstract away inner details

Reply:
Removed